internal/database/sqlcommon/tokenpool_sql.go

```diff
@@ -96,6 +97,7 @@ func (s *SQLCommon) UpsertTokenPool(ctx context.Context, pool *fftypes.TokenPool
 			return err
 		}
 	} else {
+		pool.Created = fftypes.Now()
```

Comment:
Where is the appropriate layer to set `created` timestamps? It seems to vary across different objects, but here in the Upsert method felt most logical/efficient to me.

Reply:
Looks appropriate to me 👍